### PR TITLE
solve the icon issue in CookieAlert

### LINKED ISSUE #274: fix close icon icon on cookies bar
flow
- go to https://staging.fightpandemics.work/
- will appear a cookies bar info on the bottom
- close icon is not very visible

Screenshot 2020-05-10 at 03 13 26

iMac, Chrome

### PR DESCRIPTION
**Pull request by harithzamri**
closes #274

### CODE REVIEW

File: client/src/components/CookieAlert.js [hunk 1]
```diff
@@ -30,12 +30,12 @@ const CookieLink = styled.a`
   text-decoration: underline;
 `;
 
-const CloseButton = styled.button`
-  align-self: start;
-`;
+// const CloseButton = styled.button`
```
Comment (anniegiang) [inline client/src/components/CookieAlert.js:33]: Let's remove the comments

File: client/src/components/CookieAlert.js [hunk 2]
```diff
@@ -62,9 +62,7 @@ export default () => {
         Your use of FightPandemics' Products is subject to these policies and
         terms.
       </div>
-      <CloseButton onClick={hideMessage}>
-        <SvgIcon src={closeX} />
-      </CloseButton>
+      <SvgIcon src={closeX} onClick={hideMessage} />
```
Comment (anniegiang) [inline client/src/components/CookieAlert.js:65]: Let's give it a `cursor: pointer`

**Comment by anniegiang** [comment]
Is there a link to the design for this?

**Comment by nadinejuraschek** [comment]
The footer is commented out for some reason. The cookie alert should display over the footer at all times.

**Comment by anniegiang** [comment]
> The footer is commented out for some reason. The cookie alert should display over the footer at all times, I think it does that already.
> 
> The design can be found in the Design System MVP on the right side.

Ok good to know! We should probably get the Footer to display again..

**Comment by anniegiang** [comment]
Ok awesome thanks for making the changes. 

I've realized there's another issue related to this but I think it's out of scope for this PR. I'm going to open another issue regarding it.

**Pull request merged**